File: 프로그래머스/lv1/12915. 문자열 내 마음대로 정렬하기/문자열 내 마음대로 정렬하기.js
function solution(strings, n) {
    return strings.sort((a, b) => {
        if (a.charCodeAt(n) < b.charCodeAt(n)) return -1;
        if (a.charCodeAt(n) > b.charCodeAt(n)) return 1;
        
        return a < b ? -1 : 1;
    });

}

File: 프로그래머스/1/12915. 문자열 내 마음대로 정렬하기/문자열 내 마음대로 정렬하기.py
def solution(strings, n):
    return sorted(strings, key=lambda x: (x[n], x))
    # key 함수 : 리스트의 각 요소에 대해 호출되며, 반환값이 정렬의 기준이 됨
    # 첫 번째 요소가 동일한 경우, 두 번째 요소를 기준으로 정렬함